crates/sui-storage/src/event_store/sql.rs

```diff
@@ -329,28 +329,28 @@ impl From<SqliteRow> for StoredEvent {
 }
 
 const TS_QUERY: &str =
-    "SELECT * FROM events WHERE timestamp >= ? AND timestamp < ? ORDER BY timestamp DESC LIMIT ?";
+    "SELECT * FROM events WHERE timestamp >= ? AND timestamp < ? ORDER BY timestamp ASC LIMIT ?";
```

Comment:
I'm fine changing these to ASC if most users would prefer increasing time order.   Can you please update the RustDoc in `event_store/mod.rs` in the Trait though?  Thanks

Reply:
Sure can!

Reply:
Done!